code/controllers/subsystem/vote.dm

```diff
@@ -47,7 +49,33 @@ SUBSYSTEM_DEF(vote)
 				client_popup.open(0)
 			next_pop = world.time+VOTE_COOLDOWN
 
-
+/**
+  * Renders a statpanel. Directly uses statpanel/stat calls since this is called from base of mob/Stat().
+  */
+/datum/controller/subsystem/vote/proc/render_statpanel(mob/M)
+	if(!mode)		// check if vote is running
+		return
+	if(!statpanel("Status"))		// don't bother if they're not focused on this panel
+		return
+	var/static/list/supported = list(PLURALITY_VOTING, APPROVAL_VOTING)
+	stat("Vote active!", "There is currently a vote running. Question: [question]")
+	if(!(vote_system in supported))
+		stat("<STATPANEL VOTING DISABLED>", "The current vote system is not supported by statpanel rendering. Please vote manually by opening the vote popup using the action button or chat link.")
+		return
+	stat("Time Left:", "[round(end_time - world.time)] seconds")
+	stat(null, null)
+	stat("Choices:", null)
+	stat(null, null)
+	for(var/i in 1 to choice_statclicks.len)
+		var/choice = choice_statclicks[i]
+		var/ivotedforthis = FALSE
+		switch(vote_system)
+			if(APPROVAL_VOTING)
+				ivotedforthis = voted[usr.ckey] && (i in voted[usr.ckey])
+			if(PLURALITY_VOTING)
+				ivotedforthis = voted[usr.ckey] == i
+		stat(ivotedforthis? "\[X\]" : "\[ \]", choice_statclicks[choice])
+	stat(null, null)
```

Comment:
```suggestion
	stat(null, "Vote active! There is currently a vote running. Question: [question]")
	if(!(vote_system in supported))
		stat(null, "<STATPANEL VOTING DISABLED> The current vote system is not supported by statpanel rendering. Please vote manually by opening the vote popup using the action button or chat link.")
		return
	stat(null, "Time Left: [round(end_time - world.time)] seconds")
	stat(null, null)
	stat(null, "Choices:")
	stat(null, null)
	for(var/i in 1 to choice_statclicks.len)
		var/choice = choice_statclicks[i]
		var/ivotedforthis = FALSE
		switch(vote_system)
			if(APPROVAL_VOTING)
				ivotedforthis = voted[usr.ckey] && (i in voted[usr.ckey])
			if(PLURALITY_VOTING)
				ivotedforthis = voted[usr.ckey] == i
		stat(null, "[ivotedforthis ? "\[X\]" : "\[ \]"] [choice_statclicks[choice]]")
	stat(null, null)
```
status statpannel is (null, thing)!

Reply:
cant do that with how statclicks works as that'll turn it into a text expression which can't be click()ed